reconcile/utils/jinja2/utils.py

```
@@ -397,7 +397,7 @@ def process_jinja2_template(
     })
     if "_template_mocks" in vars:
         for k, v in vars["_template_mocks"].items():
-            vars[k] = lambda *args, **kwargs: v  # noqa: B023
+            vars[k] = lambda *args, **kwargs: v  # ruff: ignore[function-uses-loop-variable]
```

Ruff (0.16.0)

[warning] Function definition does not bind loop variable `v`

(B023)